src/Elastic.Transport/Requests/IRequestParameters.cs

```diff
@@ -50,7 +44,7 @@ public interface IRequestParameters
 		/// <summary>
 		/// Gets the stringified representation of a query string value as it would be sent to Elasticsearch.
 		/// </summary>
-		string GetResolvedQueryStringValue(string n, ITransportConfigurationValues s);
+		string GetResolvedQueryStringValue(string n, ITransportConfiguration s);
```

Comment:
Unrelated to this PR but I'm mentioned it here since I noticed it! Should we give these params descriptive names?

```suggestion
		string GetResolvedQueryStringValue(string name, ITransportConfiguration transportConfiguration);
```